src/main/js/helpers/constants.js

```diff
@@ -12,35 +12,41 @@ import * as DEFAULT_SHAPES from "../core/Shape/shapeDefinitions";
  *
  * @public
  * @property {string} BLACK "#1c1f21"
- * @property {string} BLUE "#007cc3"
- * @property {string} GREEN "#78c346"
- * @property {string} LIGHT_PURPLE "#c985da"
- * @property {string} ORANGE "#ffc20a"
- * @property {string} YELLOW "#ffc20a"
+ * @property {string} BLUE "#176ba0"
+ * @property {string} GREEN "#076e00"
  * @property {string} GREY "#a7aaab"
- * @property {string} LIGHT_BLUE "#26a2e5"
- * @property {string} LIGHT_GREEN "#a5d784"
- * @property {string} PURPLE "#78288c"
- * @property {string} LIGHT_ORANGE "#ff7d00"
+ * @property {string} LIGHT_BLUE "#008fe0"
+ * @property {string} LIGHT_GREEN "#4b9b1c"
+ * @property {string} LIGHT_ORANGE "#e46511"
+ * @property {string} LIGHT_PINK "#e16092"
+ * @property {string} LIGHT_PURPLE "#bc70c7"
  * @property {string} LIGHT_YELLOW "#ffda6c"
+ * @property {string} ORANGE "#b54900"
+ * @property {string} PINK "#ae0868"
+ * @property {string} PURPLE "#9c19a3"
  * @property {string} WHITE "#ffffff"
+ * @property {string} YELLOW "#ffc20a"
  * @enum {{Object: string}}
  */
+
 export const COLORS = {
     BLACK: "#1c1f21",
-    BLUE: "#007cc3",
-    GREEN: "#78c346",
-    LIGHT_PURPLE: "#c985da",
-    ORANGE: "#ed5e00",
-    YELLOW: "#ffc20a",
+    BLUE: "#176ba0",
+    GREEN: "#076e00",
     GREY: "#a7aaab",
-    LIGHT_BLUE: "#26a2e5",
-    LIGHT_GREEN: "#a5d784",
-    PURPLE: "#78288c",
-    LIGHT_ORANGE: "#ff7d00",
+    LIGHT_BLUE: "#008fe0",
+    LIGHT_GREEN: "#4b9b1c",
+    LIGHT_ORANGE: "#e46511",
+    LIGHT_PINK: "#e16092",
+    LIGHT_PURPLE: "#bc70c7",
     LIGHT_YELLOW: "#ffda6c",
-    WHITE: "#ffffff"
+    ORANGE: "#b54900",
+    PINK: "#ae0868",
+    PURPLE: "#9c19a3",
+    WHITE: "#ffffff",
+    YELLOW: "#ffc20a"
 };
```

Comment:
Nit: I would alphabetize these similar to how I mentioned above, to keep it consistent with the documentation.

Reply:
nit: We can update it
```js
    // Dark colors
    BLUE: "#176ba0",
    GREEN: "#076e00",
    ORANGE: "#b54900",
    PINK: "#ae0868",
    PURPLE: "#9c19a3",
    YELLOW: "#ffc20a"

    // Light colors
    LIGHT_BLUE: "#008fe0",
    LIGHT_GREEN: "#4b9b1c",
    LIGHT_ORANGE: "#e46511",
    LIGHT_PINK: "#e16092",
    LIGHT_PURPLE: "#bc70c7",
    LIGHT_YELLOW: "#ffda6c",

   // Common colors
    BLACK: "#1c1f21",
    GREY: "#a7aaab",
    WHITE: "#ffffff",
```

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/214/files#diff-965ff141b3e1c375076dd23d7144dd8aL29)